scripts/extensions/datetimeField/src/getToggleTemplateDateTimeField.tsx

```diff
@@ -8,34 +8,45 @@ interface IPropsAdditional {
 }
 
 export function getToggleDateTimeField(superdesk: ISuperdesk) {
-    const {notify} = superdesk.ui;
     const {gettext} = superdesk.localization;
 
     return class ToggleDateTimeField
         extends React.PureComponent<IEditorComponentProps<string | null, IDateTimeFieldConfig> & IPropsAdditional> {
         render() {
+            const initialConfig = this.props.config.initial_offset_minutes;
             const checkbox = (
                 <Switch
                     value={this.props.value != null}
                     onChange={(value) => {
                         if (value) {
-                            const initialConfig = this.props.config.initial_offset_minutes;
-
                             this.props.setValue(`{{ now|add_timedelta(minutes=${initialConfig})|iso_datetime }}`);
-                            notify.success(gettext('Time offset is configured to be {{minutes}} minutes',
-                                {minutes: initialConfig},
-                            ));
                         } else {
                             this.props.setValue(null);
                         }
                     }}
                 />
             );
+            const messageText = gettext(
```

Comment:
It'd be better to use placeholders that don't repeat the words around it.

Instead of `offset is {{offset}}` or `configured to {{minutes}} minutes`, prefer

`offset is {{x}}`, `configured to {{y}} minutes and {{z}} seconds`

It will be clearer for translators this way.

I have noticed that it was not possible to use the same placeholder multiple times in the same translation and pushed a commit to your branch to fix this.

Reply:
@GyanP I see you forgot to address this one. Can you change the placeholder to `x`, `n` or something similar to avoid the same words("minutes") next to each other? (`configured to be {{minutes}} minutes`)